core/src/avm1/globals/system.rs

```diff
@@ -0,0 +1,544 @@
+use crate::avm1::function::Executable;
+use crate::avm1::object::Object;
+use crate::avm1::property::Attribute::{DontDelete, DontEnum, ReadOnly};
+use crate::avm1::return_value::ReturnValue;
+use crate::avm1::{Avm1, Error, ScriptObject, TObject, Value};
+use crate::context::UpdateContext;
+use enumset::{EnumSet, EnumSetType};
+use gc_arena::MutationContext;
+use num_enum::TryFromPrimitive;
+use std::convert::TryFrom;
+
+/// Available type of sandbox for a given SWF
+pub enum SandboxType {
+    Remote,
+    LocalWithFile,
+    LocalWithNetwork,
+    LocalTrusted,
+}
+
+impl SandboxType {
+    pub fn get_sandbox_name(&self) -> &str {
+        match self {
+            SandboxType::Remote => "remote",
+            SandboxType::LocalWithFile => "localWithFile",
+            SandboxType::LocalWithNetwork => "localWithNetwork",
+            SandboxType::LocalTrusted => "localTrusted",
+        }
+    }
```

Comment:
Wonder if most of these should implement `Display` (and thus `ToString`), and then you can use `to_string`. `Manufacturer` is the only one that has multiple formats.